doc/nrf/ug_matter_device_certification.rst

```diff
@@ -7,44 +7,110 @@ Matter certification
+
+Before you start applying for the Matter certification, you must join CSA, as detailed in the following table.
+
+.. table::
+
+   +-------------------------------+---------------------------+-----------------------------+----------------------------------------+
+   | Technology certification      | Stage for certification   | SDO to join                 | Minimum SDO membership level required  |
+   +===============================+===========================+=============================+========================================+
+   | Matter Certification          | Production                | `Join CSA`_                 | Adopter                                |
+   +-------------------------------+---------------------------+-----------------------------+----------------------------------------+
+
+Joining CSA allows you to meet the following requirements:
+
+* Familiarity with the Matter specification, which is the standard for the certification.
+  You can access the specification from the `Matter Resource Kit`_ after you join CSA.
+* Assignment of Vendor ID (VID), which is required when applying for certification.
+  You can apply to `CSA's Certification Team`_ to obtain VID.
+
+.. _ug_matter_device_certification_testing:
+
+Testing
+=======
+
+The Matter certification process requires testing to be performed at an Authorized Test Laboratory (ATL), using a test plan developed by CSA.
+You can find the list of available ATLs on the `CSA's Testing Providers`_ page.
+
+Each ATL has its own submission process, but all require a Declaration of Conformity and a Protocol Implementation Conformance Statement (PICS).
+Both documents confirm that the Matter component you are going to certify is meeting the protocol requirements.
+
+The ATL can contact you if your Matter component is failing any of the test cases.
+
+.. _ug_matter_device_certification_reporting:
+
+Reporting
+=========
+
+After the Matter component successfully passes all required test cases, the ATL sends a test report to both you and CSA.
+The test report summarizes the testing process and includes a variety of information, including details about the device, `Certification type`_, test plans, and Test Harness.
+
+Application
+===========
+
+Once you have obtained the test report for your Matter component, you can apply for the certification to CSA.
+You can do this online from the `CSA Member Resources page`_ using the different Certification and Testing tools to submit the required documentation.
+
+Application requirements
+------------------------
+
+The final application for certification requires you to send the following documents to CSA:
 
-To read more about Matter and dependent certification, get familiar with CSA's Certification Policy and Procedures document.
+* Declaration of Conformity and `Matter Attestation of Security`_ (see below)
+* PICS
+* Description of the Matter component to be certified (and photos of the end product, for the product certification)
```

Comment:
Also 
Security Matter Attestation 
and 
Dependent Transport Attestation
you can find them here: https://groups.csa-iot.org/wg/members-all/document/folder/2255